ExportHandlers.md

```diff
@@ -0,0 +1,253 @@
+# Handling of Text, Lines and Symbols
+
+Text and Lines have in common that there are a multitude of different styles, and users can add their own styles.  The style of a text or line object can be determined by looking at their `StyleId` and/or `StyleAsText` properties.  Each style should be exported in a different fashion.
```

Comment:
I know it's a habit of yours to put a double space after a period. I don't care too much, but at least it should be consistent. In some cases there is only a single space.